templates/traefik.yaml

```
@@ -6,6 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: traefik
     kubeaddons.mesosphere.com/provides: ingresscontroller
+  annotations:
+    kubeaddons.mesosphere.io/appVersion: "1.68.4"
+    kubeaddons.mesosphere.io/endpoint: "/ops/portal/traefik"
```

Natalie requested that we use the names on the endpoints, ie
```suggestion
    kubeaddons.mesosphere.io/endpoint/traefik: "/ops/portal/traefik"
```